napari/_qt/_qapp_model/qactions/_window.py

```diff
@@ -0,0 +1,84 @@
+"""Actions related to the 'Window' menu that require Qt."""
+
+from typing import List
+
+from app_model.types import Action, ToggleRule
+
+from ...._app_model.constants import CommandId, MenuGroup, MenuId
+from ....utils.translations import trans
+from ...qt_main_window import Window
```

Comment:
Could we use absolute path in imports? The four dots are not readable.

Reply:
I tend to agree but I am not sure about what is the standard in napari, from looking at the code base it seems that modules use relative and tests use absolute so I followed?